string type
int64 duration

---

string type